hibernate-core/src/main/java/org/hibernate/query/QueryProducer.java

```diff
@@ -50,7 +50,7 @@
 	 * Create a typed {@link Query} instance for the given HQL/JPQL query string.
 	 *
 	 * @param queryString The HQL/JPQL query
-	 *
+	 * @param resultClass The type of the query result
```

Comment:
The above is to fix a missing Javadoc element along the way, not directly related to the ticket.